oak/server/rust/oak_runtime/src/introspect.rs

```diff
@@ -145,11 +145,11 @@ async fn make_server(
     runtime: Arc<Runtime>,
     termination_notificiation_receiver: tokio::sync::oneshot::Receiver<()>,
 ) {
-    // Construct our SocketAddr to listen on...
-    let addr = SocketAddr::from(([127, 0, 0, 1], port));
+    // Initialize SocketAddr to listen on.
+    let addr = SocketAddr::from(([0, 0, 0, 0, 0, 0, 0, 0], port));
```

Comment:
There must be a better way of initializing this, perhaps https://doc.rust-lang.org/std/net/struct.Ipv6Addr.html#associatedconstant.LOCALHOST ?

Reply:
Done